### ISSUE TITLE
MsgPack incorrectly serializes longs

### ISSUE THREAD

**Issue by philipcass** [msgpack/msgpack-python#114]
msgpack-python seems to get confused when serializing longs < 4,294,967,295.

```
>>> msgpack.unpackb(msgpack.packb(4294967295L))
-1
```

Found in msgpack-python 0.4.2

**Comment by methane** [comment]
@philipcass Do you use Windows?  Is your Python 32bit or 64bit?

**Comment by philipcass** [comment]
Ubuntu 12.04 LTS 32bit

**Comment by methane** [comment]
I can't reproduce it on 32bit Python on Mac.
Could you try following?

```
>>> import msgpack
>>> msgpack.packb(4294967295L)
'\xce\xff\xff\xff\xff'
>>> msgpack.unpackb('\xce\xff\xff\xff\xff')
4294967295L
>>> msgpack.version
(0, 4, 2)
>>> msgpack._packer
<module 'msgpack._packer' from 'msgpack/_packer.so'>
```

**Comment by methane** [comment]
Oh, it's my fault. I've tested HEAD.

7f623c09060fd2561efaa6e5b4ef40da9747ec9e has fixed it already.  I'll release new version in a few days.

**Comment by philipcass** [comment]
Awesome! Thanks for your work!

**Comment by methane** [comment]
I've released 0.4.3.

**Comment by ThomasWaldmann** [comment]
Duplicate of #101.
